hack/version.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# Copyright 2014 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# (Loosely) used https://github.com/kubevirt/kubevirt/blob/master/hack/version.sh as a template
+
+# Grovels through git to return build version.
+source hack/common.sh
+
+function ovscni::version::get_version_vars() {
+    # If the source was exported through git archive, then # we likely don't
+    # have a git tree, but these magic values may be filled in.
+    if [[ '$Format:%%$' == "%" ]]; then
```

Comment:
I'm sorry, i will need a little bit of help here. What does this line check for?

Reply:
Not actually exactly sure. Leaving it intact in the new version where we copy paste the function as-is from kubevirt (works now that v0.1.0 is tagged). I hope it's ok to leave it as-is as long as it's close to kubevirt.